applications/StructuralMechanicsApplication/tests/cpp_tests/test_truss.cpp

```diff
@@ -379,6 +379,46 @@ namespace Kratos::Testing
         constexpr auto expected_stress = induced_strain * youngs_modulus;
         KRATOS_EXPECT_DOUBLE_EQ(expected_stress, stress_vector[0][0]);
 
+        constexpr auto pre_stress = 1.0e5;
+        p_element->GetProperties().SetValue(TRUSS_PRESTRESS_PK2, pre_stress);
+        p_element->CalculateOnIntegrationPoints(PK2_STRESS_VECTOR, stress_vector, r_process_info);
+        KRATOS_EXPECT_DOUBLE_EQ(expected_stress + pre_stress, stress_vector[0][0]);
+    }
+KRATOS_TEST_CASE_IN_SUITE(LienarTrussElement2D_CalculatesPK2Stress, KratosStructuralMechanicsFastSuite)
```

Comment:
There is a minor typo in the test name:
```suggestion
KRATOS_TEST_CASE_IN_SUITE(LinearTrussElement2D_CalculatesPK2Stress, KratosStructuralMechanicsFastSuite)
```

Reply:
I see this was already done